### PR TITLE
MWPW-151549: Modify blog playwright test for bacom

### PR DESCRIPTION
**Pull request by JasonHowellSlavin**
- Modifies scripts created for opening docx and modifying on bacom-blog for use on bacom
- Adds a step to utilize hlx sign in to ensure authentication 

Resolves: [MWPW-151549](https://jira.corp.adobe.com/browse/MWPW-151549)

### CODE REVIEW

File: configs/bacom.config.js [hunk 1]
```diff
@@ -30,8 +30,7 @@ const config = {
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
   reporter: process.env.CI
     ? [['github'], ['../utils/reporters/json-reporter.js'], ['../utils/reporters/json-reporter.js']]
-    : [['html', { outputFolder: 'test-html-results', open: 'never' }],
-      ['list'], ['../utils/reporters/base-reporter.js']],
+    : [['html', { outputFolder: 'test-html-results', open: 'never' }]],
```
[the remark this answers is not in the document]
Comment (JasonHowellSlavin) [reply configs/bacom.config.js]: I'll add it back, might have been a mistake.

File: configs/bacom.config.js [hunk 2]
```diff
@@ -29,9 +29,11 @@ const config = {
   workers: process.env.CI ? 2 : undefined,
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
   reporter: process.env.CI
-    ? [['github'], ['../utils/reporters/json-reporter.js'], ['../utils/reporters/json-reporter.js']]
-    : [['html', { outputFolder: 'test-html-results', open: 'never' }],
-      ['list'], ['../utils/reporters/base-reporter.js']],
+    ? [['github'], ['list'], ['../utils/reporters/base-reporter.js']]
```
Comment (JackySun9) [inline configs/bacom.config.js:32]: @Dli3 you already fixed this, right?

File: configs/bacom.config.js [hunk 3]
```diff
@@ -47,6 +46,27 @@ const config = {
 
   /* Configure projects for major browsers */
   projects: [
+    {
```
[the remark this answers is not in the document]
Comment (JackySun9) [reply configs/bacom.config.js:52]: @Dli3 please check this

File: configs/bacom.config.js (hunk 2 above)
Comment (Dli3) [reply configs/bacom.config.js:32]: Yeah, I updated it in https://github.com/adobecom/nala/pull/377 .

File: configs/bacom.config.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (Dli3) [reply configs/bacom.config.js:52]: @JasonHowellSlavin We shouldn't have the bacom_page env here under projects, or else the tests will run against page.

**Comment by JasonHowellSlavin** [comment]
Hi @JackySun9 and @Dli3 , 

I had to rebase, which is why there were some weird changes in the commits. Should all be good now. Please take another look! Thank you both for your help.

**Review by Dli3 — APPROVED** [review]

**Review by JackySun9 — APPROVED** [review]

**Pull request merged**